src/services/kafka/producer.ts

```diff
@@ -1,37 +1,54 @@
 
-import { Kafka } from 'kafkajs'
+import { Kafka, logCreator, logLevel, Producer, ProducerBatch } from 'kafkajs'
```

Comment:
Mostly because we have a different style guide where I work so I got used to it. Anyway I think that makes sense and since I'll be using the airbnb style guide, I'll change this